superset/connectors/sqla/models.py

```diff
@@ -756,9 +765,6 @@ def values_for_column(
         if limit:
             qry = qry.limit(limit)
 
-        if not contain_null:
-            qry = qry.where(target_col.get_sqla_col().isnot(None))
-
```

Comment:
These codes were introduced by this [PR](https://github.com/apache/superset/pull/17168). After we provide **replacing value** mechanism, this snippet is unused.